docs/firebase-emulators.md

```diff
@@ -0,0 +1,43 @@
+### Firebase Emulators
+
+Instead of running just the functions' emulator, the full suite of emulators can be used.
+You can find out more at https://firebase.google.com/docs/emulator-suite.
+
+**Note**: If your code accidentally invokes non-emulated (production) resources, there is a chance of data change, usage and billing.
+To prevent this, you might opt in to use a Firebase project name beginning with `demo-` (e.g. `demo-staging`) in which case no production resources will be used.
+
+First though, a **major version of 8** for the Firebase client is required:
+```bash
+npm i firebase@8
+```
```

Comment:
I'm not actually sure if this is true, or if it is, why. The PR that changed the docs for this got merged before it was ready/reviewed/approved by me. We should really check this out further and ensure it's correct. (that goes for all these using emulators docs)

Reply:
you're right, it worked with a simple npx firebase ( latest version ) 
removed in fb84b99